docs/data-storage.md

```diff
@@ -1,80 +1,38 @@
-save data will be ignored and any attempt to retrieve data will 
-return the global settings with default values.
-
-Games also adhere to the BBC 'Functional' Cookie so a user can opt out of these features. 
-Users can manage preferences here: [Manage Cookie Settings (Live)].
-
-*All games must be prepared to handle this scenario.*
+# Using Local Storage
 
 ## Saving data
 
-````
-gmi.setGameData(key, value);
-````
+Ideally, data should not be stored. If it is stored, data must not be personal and should adhere to the cookie policy of the browser. If the game contains sign in, it must respect the privacy settings on the user's account.
 
-Stores a JSON key-value pair that can then be retrieved with getAllSettings().
+Data must be saved through the [GMI](gmi.md). This ensures all data access complies with BBC data policies.
 
-The exceptions to this are the [global settings](gmi.md#global-game-settings): audio, subtitles, and motion:
+```javascript
+gmi.setGameData(key, value);
+```
 
-````
-gmi.setAudio(true/false)
-gmi.setSubtitles(true/false)
-gmi.setMotion(true/false)
-````
+Stores a JSON key-value pair that can then be retrieved with `getAllSettings()`.
 
 ## Loading data
 
-````
+```javascript
 gmi.getAllSettings();
-````
-
-Returns a JSON object with [global settings](gmi.md#global-game-settings) and specific gameData.
-
-Specific game data is stored under gameData:
-
-````
-gmi.getAllSettings().gameData
-````
-
-This object will contain all of the properties set by this game using setGameData.
-
-[Global settings](gmi.md#global-game-settings) can be accessed like so:
-
-````
-gmi.getAllSettings().audio
-gmi.getAllSettings().subtitles
-gmi.getAllSettings().motion
-````
-
-## Example
-[Code Example](../src/main.js) - open [index.html](../src/index.html) in your browser to see this in action.
+```
 
-This is also available to see in [CAGE](https://www.bbc.co.uk/cbeebies/embed/game/childrens-games-starter-pack).
+Returns a JSON object with [global settings](settings.md#global-settings) and specific game data.
 
-## Testing
+Specific game data is stored under `gameData`:
 
-### CAGE
-Change your 'Functional' cookie preferences at [Manage Cookie Settings (Live)] to see the [CAGE demo](https://www.bbc.co.uk/cbeebies/embed/game/childrens-games-starter-pack)
-behave differently.
+```javascript
+gmi.getAllSettings().gameData;
+```
 
-### Local
-Edit your index.html so that gmi.areCookiesAllowed() returns true/false to observe how your game behaves. 
+This object will contain all of the properties set by this game using `setGameData`.
 
+[Global settings](settings.md#global-settings) can be accessed like so:
 
-[Manage Cookie Settings (TEST)]: http://www.test.bbc.co.uk/privacy/cookies/managing/cookie-settings
-[Manage Cookie Settings (LIVE)]: http://www.bbc.co.uk/privacy/cookies/managing/cookie-settings
+```javascript
+gmi.getAllSettings().audio;
+gmi.getAllSettings().motion;
+gmi.getAllSettings().subtitles;
 
-* [Home](../README.md)
```

Comment:
Did you mean to delete this?

Reply:
This was just because i tried to use the same document across all our repos for consistency ill add the links back in